### DISCUSSION TITLE
Can "MagicLink signedUp user" set password?

### DISCUSSION THREAD

**Discussion by raja-ilayaperumal** [nhost/nhost#2145] [General]
We have set of user's who signed up using magic link. Now we want to enable email with password sign-up/sign-in method.

Old user tried to sign-up with email and password, but it throws error "Email already in use". During sign-in, it throws, "Password mismatch"

How to allow existing user's to set password for their account?

**Comment by dbarrosop** [comment]
A user can't sign up twice with the same email, try with changePassword

**Reply by raja-ilayaperumal** [reply]
Yes. I found that, we can send resetPassword link to re-login and changePassword.
